ekm-over-vpc-onboarding/README.md

```diff
@@ -79,3 +80,7 @@ This guide provides instructions of an automation for Cloud External Key Manager
 | vpc\_project\_id | ID of the VPC project |
 
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+**Note 3:** You will need to manually set crypto-key-version to `primary version` by either using Google console or Google CLI command. 
```

Comment:
Maybe let's add a sample command? Or a link to https://cloud.google.com/sdk/gcloud/reference/kms/keys/set-primary-version

Reply:
Good idea, sure.